_data/events.yml

```diff
@@ -224,19 +224,78 @@
     title: View meetup event
     target: _blank
 
-- title: Coding Club with Women Coding Community
+- title: |
+    Coding Club with Women Coding Community
   description: |
-    Learn Python 101: In subsequent sessions, we will delve into solving basic algorithmic problems, using "Grokking Algorithms" as our reference book, and thoroughly cover the fundamentals of the language.
-  category_style: coding-club
-  category_name: Coding Club
-  date: WED, SEP 05, 2024
+    We are excited to announce our next Coding Club session as a part of Women Coding Community!
+  category: Online Event
+  category_name: Online Event
+  date: Thu, Sep 5, 2024
   expiration: "20240905"
-  speaker: "Nonna Shakhova, Cloud Data Engineer"
-  time: 7:00 PM to 8:00 PM BST
+  host: "Nonna Shakhova"
+  speaker: "Liliia Rafikova"
+  time: 6:00 PM UTC
   image:
-    path: "/assets/images/events/event-coding-club-3.jpg"
-    alt: square poster of event
+    path: "/assets/images/events/event_522666074.webp"
```

Comment:
If possible, I think we may want to also update the assets naming to match, but not really a problem though.

Coding club and Writing club also use the same banner for their recurring events because it's undated, so it doesn't have to be different banners for different events

Reply:
Good point: @sjanagill could you make this adjustment in the script?

Reply:
Done. Please check https://github.com/Women-Coding-Community/WomenCodingCommunity.github.io/pull/331/